pandas/tests/indexes/multi/test_duplicates.py

```diff
@@ -264,3 +241,84 @@ def test_get_duplicates():
 
             tm.assert_numpy_array_equal(mi.duplicated(),
                                         np.zeros(len(mi), dtype='bool'))
+
+
+@pytest.mark.parametrize('keep, expected', [
+    ('first', np.array([False, False, False, True, True, False])),
+    ('last', np.array([False, True, True, False, False, False])),
+    (False, np.array([False, True, True, True, True, False]))
+])
+def test_duplicated(idx_dup, keep, expected):
```

Comment:
Was there a change to this and subsequent test? Unclear why this is showing in diff